src/test/java/com/dakar/dakar/acceptance/config/DataTableConfigurer.java

```diff
@@ -0,0 +1,34 @@
+package com.dakar.dakar.acceptance.config;
+
+import com.dakar.dakar.models.Journey;
+import cucumber.api.TypeRegistry;
+import cucumber.api.TypeRegistryConfigurer;
+import io.cucumber.datatable.DataTableType;
+import io.cucumber.datatable.TableEntryTransformer;
+
+import java.util.Locale;
+import java.util.Map;
+
+/*
+ * Maps datatables in feature files to custom domain objects.
+ */
+public class DataTableConfigurer implements TypeRegistryConfigurer {
```

Comment:
I think I don't understand why we need this.
We can discuss it during the next call

Reply:
I just tried something new.

Reply:
really interesting ! I learned something.